tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,368 @@
+# the integer IDs to experimental conditions or events. In this dataset, the
+# mapping looks like this:
 #
-# Define epochs parameters:
-
-event_id = dict(aud_l=1, aud_r=2)  # event trigger and conditions
-tmin = -0.2  # start of each epoch (200ms before the trigger)
-tmax = 0.5  # end of each epoch (500ms after the trigger)
-
-##############################################################################
-# Exclude some channels (original bads + 2 more):
-
-raw.info['bads'] += ['MEG 2443', 'EEG 053']
-
-##############################################################################
-# The variable ``raw.info['bads']`` is just a Python list.
+# +----------+----------------------------------------------------------+
+# | Event ID | Condition                                                |
+# +==========+==========================================================+
+# | 1        | auditory stimulus (tone) to the left ear                 |
+# +----------+----------------------------------------------------------+
+# | 2        | auditory stimulus (tone) to the right ear                |
+# +----------+----------------------------------------------------------+
+# | 3        | visual stimulus (checkerboard) to the left visual field  |
+# +----------+----------------------------------------------------------+
+# | 4        | visual stimulus (checkerboard) to the right visual field |
+# +----------+----------------------------------------------------------+
+# | 5        | smiley face (catch trial)                                |
+# +----------+----------------------------------------------------------+
+# | 32       | subject button press                                     |
+# +----------+----------------------------------------------------------+
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data; the ``/`` character in the dictionary keys allows pooling
+# across conditions by requesting partial condition descriptors (i.e.,
+# requesting ``'auditory'`` will select all epochs with Event IDs 1 and 2;
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected). Here we'll also make use of the :class:`~mne.Info`
+# attribute to get the sampling frequency of the recording (so our x-axis will
+# be in seconds instead of in samples).
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
```

Comment:
Either in **Detecting experimental events** or **Epoching continuous data** I would point `mne.make_fixed_length_events`. WDYT?